python/paddle/fluid/tests/unittests/op_test.py

```diff
@@ -351,13 +357,132 @@ def _calc_output(self, place, parallel=False, no_check_set=None, loss=None):
         # fetch_list = map(block.var, fetch_list)
         if not isinstance(fetch_list[0], fluid.framework.Variable):
             fetch_list = list(map(block.var, fetch_list))
+
+        if enable_inplace is not None:
+            build_strategy = fluid.BuildStrategy()
+            build_strategy.enable_inplace = enable_inplace
+            build_strategy.memory_optimize = False
+
+            compiled_prog = fluid.CompiledProgram(program).with_data_parallel(
+                build_strategy=build_strategy, places=place)
+            program = compiled_prog
+
         executor = Executor(place)
         outs = executor.run(program,
                             feed=feed_map,
                             fetch_list=fetch_list,
                             return_numpy=False)
         return outs, fetch_list
```

Comment:
I think that we should fetch all forward outputs instead of `fetch_list` which is not including `no_check_set`? So that we can make backward inputs to check the accuracy of backward inplace?